platform/darwin/resources/lt.lproj/Foundation.stringsdict

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>COORD_DEG_LONG</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@degrees@</string>
+		<key>degrees</key>
+		<dict>
```

Comment:
This dict lacks `zero` key:

```plist
<key>zero</key>
<string>%d laipsnių</string>
```

And I'm also not sure about the correctness of `many` key. Can you provide me with the context? Because the value written is used for possessive noun case (ditto for others: minutes and seconds)